### PR TITLE
Resolve pool activation operations more cleanly

### LINKED ISSUE #743: Pool activation happens more than once
Token pool activation is sometimes happening twice, and only one of the operations gets resolved.

This is partly because activation operations are resolved only by TX ID, not by the actual operation ID.
https://github.com/hyperledger/firefly/blob/5afbecf2563d290f014e45b2542e037947ecd3ec/internal/events/token_pool_created.go#L68

Need to revisit this flow.

### PR DESCRIPTION
**Pull request by awrichar**
Fixes #743

### CODE REVIEW

File: internal/events/token_pool_created.go [hunk 1]
```diff
@@ -65,8 +65,6 @@ func (em *eventManager) confirmPool(ctx context.Context, pool *fftypes.TokenPool
 		return err
```
Comment (awrichar) [inline internal/events/token_pool_created.go:65]: It occurs to me that this operation query is probably not needed at all now. I think it was only ever here to do the resolve.

Comment (peterbroadhurst) [reply internal/events/token_pool_created.go:65]: Agreed. Will you remove it in this PR?

**Review by peterbroadhurst — APPROVED** [review]
Looks good to me. Approved, but with a question on whether you want to remove that now redundant DB call before merging.

File: internal/events/token_pool_created.go (hunk 1 above)
Comment (awrichar) [reply internal/events/token_pool_created.go:65]: Yes, will push a quick change to do that.

**Pull request merged**